consul/config_templates/consul-values-1.9.1-demo.yaml

```diff
@@ -0,0 +1,1841 @@
+  # feature, in case kubernetes cluster is behind egress http proxies. Additionally,
+  # it could be used to configure custom consul parameters.
+  # @type: map
+  extraEnvironmentVars: {}
+
+# Configuration for Consul servers when the servers are running outside of Kubernetes.
+# When running external servers, configuring these values is recommended
+# if setting `global.tls.enableAutoEncrypt` to true (requires consul-k8s >= 0.13.0)
+# or `global.acls.manageSystemACLs` to true (requires consul-k8s >= 0.14.0).
+externalServers:
+  # If true, the Helm chart will be configured to talk to the external servers.
+  # If setting this to true, you must also set `server.enabled` to false.
+  enabled: false
+
+  # An array of external Consul server hosts that are used to make
+  # HTTPS connections from the components in this Helm chart.
+  # Valid values include IPs, DNS names, or Cloud auto-join string.
+  # The port must be provided separately below.
+  # Note: `client.join` must also be set to the hosts that should be
+  # used to join the cluster. In most cases, the `client.join` values
+  # should be the same, however, they may be different if you
+  # wish to use separate hosts for the HTTPS connections.
+  # @type: array<string>
+  hosts: []
+
+  # The HTTPS port of the Consul servers.
+  httpsPort: 8501
+
+  # The server name to use as the SNI host header when connecting with HTTPS.
+  # @type: string
+  tlsServerName: null
+
+  # If true, consul-k8s components will ignore the CA set in
+  # `global.tls.caCert` when making HTTPS calls to Consul servers and
+  # will instead use the consul-k8s image's system CAs for TLS verification.
+  # If false, consul-k8s components will use `global.tls.caCert` when
+  # making HTTPS calls to Consul servers.
+  # **NOTE:** This does not affect Consul's internal RPC communication which will
+  # always use `global.tls.caCert`.
+  useSystemRoots: false
+
+  # If you are setting `global.acls.manageSystemACLs` and
+  # `connectInject.enabled` to true, set `k8sAuthMethodHost` to the address of the Kubernetes API server.
+  # This address must be reachable from the Consul servers.
+  # Please see the Kubernetes Auth Method documentation (https://consul.io/docs/acl/auth-methods/kubernetes).
+  # Requires consul-k8s >= 0.14.0.
+  #
+  # You could retrieve this value from your `kubeconfig` by running:
+  #
+  # ```shell
+  # kubectl config view \
+  #   -o jsonpath="{.clusters[?(@.name=='<your cluster name>')].cluster.server}"
+  # ```
+  #
+  # @type: string
+  k8sAuthMethodHost: null
+
+# Values that configure running a Consul client on Kubernetes nodes.
+client:
+  # If true, the chart will install all
+  # the resources necessary for a Consul client on every Kubernetes node. This _does not_ require
+  # `server.enabled`, since the agents can be configured to join an external cluster.
+  # @default: global.enabled
+  # @type: boolean
+  enabled: "-"
```

Comment:
Is there should be a boolean?

Reply:
`-` means that the value is inherited from `global.enabled`, which is `true` in this demo configuration